routers/biocommons_register.py

```diff
@@ -80,6 +65,62 @@ def create_user_in_db(user_data: Auth0UserData,
     return db_user
 
 
+def _notify_bundle_group_admins(
+    *,
+    bundle: BiocommonsBundle,
+    user: BiocommonsUser,
+    auth0_client: Auth0Client,
+    db_session: Session,
+    settings: Settings,
+) -> None:
+    """
+    Queue approval emails for bundle group admins when memberships require review.
+    """
+    if bundle.group_auto_approve:
+        return
+
+    membership = next(
```

Comment:
Do we really need so many guards and checks just to look up the membership we've just created? Can we just refresh by default at the start and proceed from there? I feel like we shouldn't need to write this much code every time we want to look something up in the DB.